environment.yml

```diff
@@ -6,7 +6,7 @@ channels:
 dependencies:
   - python=3.7
   - pip
-  - ctapipe=0.8.0
+  - ctapipe>=0.10.1
   - gammapy>=0.18
   - h5py
   - ipython
```

Comment:
@FrancaCassol : ctapipe_io_lst 0.6.0 is needed, right?

Reply:
We have the problem that ctapipe_io_lst cannot be on pypi, because of the rawzfitsreader dependency and you are not allowed to include dependency links in setup.py.

I am working on a solution for that. (By making protozfitsreader available on PyPI)